src/main/java/com/fasterxml/uuid/UUIDType.java

```diff
@@ -13,8 +13,10 @@ public enum UUIDType {
     NAME_BASED_MD5(3),
     RANDOM_BASED(4),
     NAME_BASED_SHA1(5),
-    UNKNOWN(0)
-    ;
+    DB_LOCALITY(6),
+    EPOCH_BASED(7),
```

Comment:
And if we follow same "standard", `TIME_BASED_EPOCH` or `TIME_BASED_FROM_EPOCH`.